sonarqube-postgresql-template.yaml

```diff
@@ -76,14 +151,16 @@ objects:
             tags: sonarqube
           from:
             kind: DockerImage
-            name: "bcgovimages/sonarqube:${SONARQUBE_VERSION}"
+            name: "docker-remote.artifacts.developer.gov.bc.ca/bcgovimages/sonarqube:${SONARQUBE_VERSION}"
```

Comment:
Should we make this a parameter, so that it is easily overridable when NOT using Artifactory?